package.json

```diff
@@ -13,7 +13,8 @@
   "files": [
     "dist/",
     "loader/",
-    "src/"
+    "src/",
+    "react"
```

Comment:
`react` or `react/`? will it make a difference?

Reply:
Good catch thank you, better safe than sorry